### PR TITLE
1458 appt notice funct on demand review report

### PR DESCRIPTION
**Pull request by MeganGeissler**
Admin Rev Report- Corrected issue with appt recert function (duplicate coding) and replaced ind code for nomi-recrt with NOMI_recrt function. 

Admin On Demand Waiver Application- Replaced appt. notice with NOMI appt to correct this issue.

### CODE REVIEW

File: admin/review-report.vbs [hunk 1]
```diff
@@ -2518,33 +2518,7 @@ ElseIf renewal_option = "Send Appointment Letters" Then
...
-						CALL write_variable_in_SPEC_MEMO(" - 1011 1st St S Hopkins 55343")
-						CALL write_variable_in_SPEC_MEMO(" - 1001 Plymouth Ave N Minneapolis 55411")
-						CALL write_variable_in_SPEC_MEMO(" - 2215 East Lake Street Minneapolis 55407")
-						CALL write_variable_in_SPEC_MEMO(" (Hours are 8 - 4:30 Monday - Friday)")
-						CALL digital_experience
-			            CALL write_variable_in_SPEC_MEMO("Domestic violence brochures are available at this website: https://edocs.dhs.state.mn.us/lfserver/Public/DHS-3477-ENG. You can always request a paper copy via phone.")
-						' CALL create_appointment_letter_notice_recertification(programs, intvw_programs, interview_end_date, last_day_of_recert)
+						CALL create_appointment_letter_notice_recertification(programs, intvw_programs, interview_end_date, last_day_of_recert)
 
 						PF4         'Submit the MEMO
```
Comment (IlseFerris) [inline admin/review-report.vbs:2523]: line 2523 - just remove the PF4. The function saves the notices so this isn't needed.

File: admin/review-report.vbs [hunk 2]
```diff
@@ -3339,26 +3313,7 @@ If renewal_option = "Send NOMIs" Then
...
-					CALL write_variable_in_SPEC_MEMO(" - 2215 East Lake Street Minneapolis 55407")
-					CALL write_variable_in_SPEC_MEMO(" (Hours are 8 - 4:30 Monday - Friday)")
-					CALL write_variable_in_SPEC_MEMO(" More detail can be found at hennepin.us/economic-supports")
-					CALL digital_experience
-					CALL write_variable_in_SPEC_MEMO("  ** If we do not hear from you by " & last_day_of_recert & "  **")
-					CALL write_variable_in_SPEC_MEMO("  **   your benefits will end on " & last_day_of_recert & ".   **")
-
+					CALL create_NOMI_recertification(caf_date_as_of_today, last_day_of_recert)
 
 					PF4         'Submit the MEMO
```
Comment (IlseFerris) [inline admin/review-report.vbs:3318]: Line 3318 - same thing, just remove the PF4

**Review by IlseFerris — CHANGES_REQUESTED** [review]
Just two little quick fixes, then go ahead and pull in. I approve thee!

**Comment by MeganGeissler** [comment]
#1036 - for updates

**Pull request merged**